base/testfiles/github-0881b.tlg

```diff
@@ -0,0 +1,195 @@
+This is a generated file for the LaTeX2e validation system.
+Don't change this file in any respect.
+Test: 0
+> \box...=
+\hbox(6.83167+0.0)x21.66138
+.\T2A/cmr/m/n/10 ^^c0
+.\T2A/cmr/m/n/10 ^^c1
+.\T2A/cmr/m/n/10 ^^c2
+! OK.
+\TEST ...\typeout {Test: #1}\sbox 0{#2}\showbox 0 
+                                                  \sbox 2{#3}\showbox 2 \use...
+l. ...     {\CYRA\CYRB\CYRV}
+> \box...=
+\hbox(6.83167+0.0)x21.66138
+.\T2A/cmr/m/n/10 ^^c0
+.\T2A/cmr/m/n/10 ^^c1
+.\T2A/cmr/m/n/10 ^^c2
+! OK.
+\TEST ...ox 0{#2}\showbox 0 \sbox 2{#3}\showbox 2 
+                                                  \usebox {0}\\$\rightarrow ...
+l. ...     {\CYRA\CYRB\CYRV}
+LaTeX Font Info:    External font `cmex10' loaded for size
+(Font)              <7> on input line ....
+LaTeX Font Info:    External font `cmex10' loaded for size
+(Font)              <5> on input line ....
+Test: 1
+> \box...=
+\hbox(6.83167+0.0)x21.66138
+.\T2A/cmr/m/n/10 ^^c0
+.\T2A/cmr/m/n/10 ^^c1
+.\T2A/cmr/m/n/10 ^^c2
+! OK.
+\TEST ...\typeout {Test: #1}\sbox 0{#2}\showbox 0 
+                                                  \sbox 2{#3}\showbox 2 \use...
+l. ...     {\CYRA\CYRB\CYRV}
+> \box...=
+\hbox(6.83167+0.0)x21.66138
+.\T2A/cmr/m/n/10 ^^c0
+.\T2A/cmr/m/n/10 ^^c1
+.\T2A/cmr/m/n/10 ^^c2
+! OK.
+\TEST ...ox 0{#2}\showbox 0 \sbox 2{#3}\showbox 2 
+                                                  \usebox {0}\\$\rightarrow ...
+l. ...     {\CYRA\CYRB\CYRV}
+Runaway argument?
+{2] {\textuppercase {\cyra \cyrb \cyrv }} {\CYRA \CYRB \CYRV } 
+! Paragraph ended before \TEST was complete.
+<to be read again> 
+                   \par 
+l. ...
```

Comment:
is that part of the intended output?

Reply:
er no and wasn't  happening here at some point a few minutes before the PR let me check...